framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/InfoPanel/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Widget, WidgetBody } from '../widget';
+import Icon from '../Icon';
+import styles from './InfoPanel.module.scss';
+
+interface Props {
+	onClick?: (event: React.MouseEvent | Event) => void;
+	color?: 'green' | 'pink' | 'yellow' | 'blue' | 'white' | 'gray' | 'red';
```

Comment:
you can use the interface exposed from Text

Reply:
yes but I think we still need to use `div` to support all the classes and props OR `Text` component can be extended if that's okay ?

Reply:
What i meant was you can do like https://github.com/LiskHQ/lisk-sdk/blob/feature/6216-add_dashboard-plugin/framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/CopiableText/index.tsx#L21-L23
or simply import and do
```
color?: TextProps['color'],
```